client/jetpack-connect/test/authorize.js

```diff
@@ -352,13 +461,22 @@ describe( 'JetpackAuthorize', () => {
 			);
 		} );
 
-		test( 'should redirect to redirect to the /jetpack/connect/plans page by default', () => {
-			const renderableComponent = <JetpackAuthorize { ...DEFAULT_PROPS } />;
-			const component = shallow( renderableComponent );
-			// component.setProps( {
-			// 	siteHasJetpackPaidProduct: true,
-			// } );
-			const target = component.instance().getRedirectionTarget();
+		test( 'should redirect to the /jetpack/connect/plans page by default', async () => {
+			renderWithRedux(
+				<JetpackAuthorize
+					{ ...DEFAULT_PROPS }
+					authQuery={ {
+						...DEFAULT_PROPS.authQuery,
+						alreadyAuthorized: true,
+					} }
+					isAlreadyOnSitesList={ true }
```

Comment:
Feel free to clean up some more of those redundant `={ true }` while we're here. There are more in the original code than my review pointed out. 😉